src/loaders/psm_load.c

```diff
@@ -57,13 +57,14 @@ static int psm_load(struct module_data *m, HIO_HANDLE *f, const int start)
 	uint32 p_ord, p_chn, p_pat, p_ins;
 	uint32 p_smp[64];
 	int type, ver /*, mode*/;
- 
+
 	LOAD_INIT();
 
 	hio_read32b(f);
 
 	hio_read(buf, 1, 60, f);
-	strncpy(mod->name, (char *)buf, 60);
+	memcpy(mod->name, (char *)buf, 60);
+	mod->name[60] = '\0';
```

Comment:
Do we want [60] or [59] here? psm.txt says that the last 60th byte should be a `^Z`

Reply:
Nice catch, I was just going off of what the original code was doing. Those should both probably be 59.

Reply:
I updated this to use 59 there instead.